case-study.Rmd

```diff
@@ -0,0 +1,467 @@
+---
+title: "ADAGE signature analysis - case study"
+author: "Jie Tan"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    fig_width: 7
+    fig_height: 7
+vignette: >
+  %\VignetteIndexEntry{User input example}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+```
+
+This script shows how we analyze the *∆anr* mutation dataset used as a case study
+in the ADAGE signature analysis paper. 
+
+# Data preparation
+Load in required libraries.
+
+```{r}
+library("ADAGEpath")
+library("DT")
+library("knitr")
+library("VennDiagram")
+library("readr")
+```
+
+Before any analysis, we need to specify the ADAGE model and the data compendium
+we want to use.
+
+```{r}
+model <- eADAGEmodel
+compendium <- PAcompendium
+probe_dist <- probedistribution
+```
+
+Let's load in the sample dataset that come with the package. It's expression data
+from *Pseudomonas aeruginosa* wild type and *∆anr* grown as biofilms on ∆F508 cystic
+fibrosis bronchial epithelial cells (CFBEs). Detailed information about
+the dataset can be found here
+[GSE67006](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67006).
+All its CEL files are stored in the folder "./inst/extdata/anr".
+To load your own dataset, simply modify this input path.
+
+```{r}
+input_path <- system.file("extdata", "anr/", package = "ADAGEpath")
```

Comment:
`system.file()` finds a file inside a package. Since this dataset is distributed with the package, users can use `system.file()` to find it rather than tracking it down in their system to find the file path themselves. The full file path on my computer is `/Library/Frameworks/R.framework/Versions/3.3/Resources/library/ADAGEpath/extdata/anr/`